plugins/servprotect.py

```diff
@@ -2,18 +2,27 @@
 from expiringdict import ExpiringDict
 
 from pylinkirc import utils
+from pylinkirc.conf import conf
```

Comment:
Import the conf module instead of the conf dict! Otherwise, rehash changes will never make this to the plugin until it is reloaded: `from pylinkirc import utils, conf`